src/VisualStudio/CSharp/Test/CodeModel/FileCodeClassTests.cs

```diff
@@ -31,6 +31,8 @@ public int A
             return a;
         }
     }
+
+    public string WindowsUserID => ""Domain""; 
```

Comment:
if was actually getting the accessor on the CodeFunction of the getter that failed so the fix should fix the setter also.